pkg/apis/options/options.go

```diff
@@ -49,6 +49,7 @@ type Options struct {
 	GitHubUsers              []string `flag:"github-user" cfg:"github_users"`
 	GitLabGroup              []string `flag:"gitlab-group" cfg:"gitlab_groups"`
 	GitlabProjects           []string `flag:"gitlab-project" cfg:"gitlab_projects"`
+	GitlabProjectAccessLevel int      `flag:"gitlab-project-access-level" cfg:"gitlab_project_access_level"`
```

Comment:
This would make the level global and limit the ability for per project level minimums.

We have several other configuration options that are complex pairs of values that use `=` as a separator.  I think that fits here e.g. `--gitlab-project some_project=20`.

In v8 structured configs, this would convert nicely to a nested hash config option.

Reply:
Unset (which many users would likely use) without an `=` would just default to the default value you want.

Reply:
If you want to keep this global, let's trim the naming down.  This flag is quite verbose.

Reply:
The `=` approach is good, thanks for the idea, implemented here https://github.com/oauth2-proxy/oauth2-proxy/pull/630/commits/29a56174d098939a97fcb0a7aef42dc44f0c7be6